pkg/migrator/writer.go

```diff
@@ -353,6 +321,183 @@ func (m *Migrator) insertOriginatorEnvelopeBlockchain(
 	return nil
 }
 
+func (m *Migrator) prepareClientEnvelope(
+	env *envelopes.OriginatorEnvelope,
+	tableName string,
+) (clientEnvelopeBytes []byte, identifier []byte, sequenceID uint64, totalSize int, err error) {
+	sequenceID = env.OriginatorSequenceID()
+	identifier = env.TargetTopic().Identifier()
```

Comment:
Curious - the sequenceID and identifier come from the function inputs (envelope). Does the function have to return them? They're known before the function call already, we could have two return values less.